src/routes/profile/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import ProfileCollections from '../../containers/ProfileCollections';
+
+import './style.css';
+
+class Profile extends Component {
+
+  componentWillMount() {
+    const httpHeaders = {
+      "method": 'GET',
+      "Content-Type": "application/json"
+    };
+    
+    // TODO: Implement dynamic user
+    fetch('http://localhost:3030/api/users/5a1da1bf972c7b073ae06e17', httpHeaders)
+      .then(res => res.json())
+      .then(user => {
+        this.props.dispatch({
+          type: "SET_CURRENT_USER",
+          payload: {user}
+        })
+      })
+      .catch(err => console.error(err));
+
+  }
+  
+  render() {
+    return (
+      <div className="tempWrapper">
+        <header className="Header">
+          <h1>Collectors Hut Header</h1>
+        </header>
+        <main className="Profile">
+          <section className="profile-header">
```

Comment:
Don't worry for now with the `sections`. Use regular `divs` and later on the project you can think about the semantic meaning of your tags. Dont waste time on this for now.